src/data/stLFR.tiny.fq

```diff
@@ -1,4 +1,4 @@
-@V10002828L1C001R013000000#543_288_92/1 1       1
+@V10002828L1C001R013000000_1 BX:Z:543_288_92
```

Comment:
From the file naming and the format before, is this supposed to be a stLFR header test?

Reply:
Did not consider that... are we talking about the barcode format differences? `#barcode/` vs `BX:Z:`? 
I'm seeing `BX:Z:` with stLFR reads too (maybe a reformatting we did on stLFR reads?) `pigz -dc /projects/btl_scratch/aafshinfard/projects/physlr/publication/datasets/stlfr/NA12878_stLFR.fq.gz | head -n1`

The reason I switched to `BX:Z:` is that the indexlr version I had installed was not able to read `#barcode/` formatted files... I'm just checking with indexlr v1.4.9 and I same behavior... 
```
 $ indexlr --bx -t16 -k8 -w1 -r tiny.bf data/stLFR.tiny.fq > new_good2.tsv
Loading repeat Bloom filter from tiny.bf
Finished loading repeat Bloom filter
indexlr 1.4.9: Using more than 5 threads does not scale, reverting to 5.
(btllib-v1.4.10) [aafshinfard@hpce706 src]$ cat new_good2.tsv
NA      11048730314773898660 4535684329547842690 9637752013492521276 8090755262123482019
```
as you see above it fails to extract the barcode... maybe somehow related to the btllib version at the time of compiling `makebf`? I don't think so but I'm just checking ...

Reply:
Yeah, I was looking at barcode format change in the header. Yes, I think if there's a "BX" on stLFR reads then it would be a reformatting that we did? 
Hmm that's funny - I took a look at the indexlr code base (in case supporting stLFR was done after the code bases split), but there appears to be logic in there for that formatting?
https://github.com/bcgsc/btllib/blob/master/include/btllib/indexlr.hpp#L400-L406

Ultimately, I guess we just have to be careful if we supported an input format before, we ideally wouldn't want to stop supporting it?

Reply:
I agree about trying to support what we used to support. How about I report this on `btllib` so we try to fix that there, and I push this for now? after that's fixed on btllib we will have to restrict the btllib version for Physlr's recipe and on README.

Reply:
Sounds fine to me!